Army/ArmyLRFull.cs

```diff
@@ -137,159 +136,94 @@ public void LR()
+        Core.Logger($"Farming {quant} Revenant's Spellscroll");
+        Bot.Quests.UpdateQuest(2060);
+        Core.RegisterQuests(6897);
+        int i = 1;
+        while (!Bot.ShouldExit && !Core.CheckInventory("Revenant's Spellscroll", quant))
         {
-            Army.AggroMonCells("Enter", "r2", "r3", "r4");
-            Army.AggroMonStart("seraphicwardage");
-            Army.DivideOnCells("Enter", "r2", "r3", "r4");
-        }
-    }
-
-    public void GetItems(string map = null, string[] Monsters = null, int questID = 000, string[] Loot = null, bool isTemp = false)
-    {
-        Core.PrivateRooms = true;
-        Core.PrivateRoomNumber = Army.getRoomNr();
-
-        Quest QuestData = Core.EnsureLoad(questID);
-        ItemBase[] RequiredItems = QuestData.Requirements.ToArray();
-        ItemBase[] QuestReward = QuestData.Rewards.ToArray();
+            Adv.BestGear(GearBoost.Undead);
+            /*Sells non-full stacks to keep in sync for each LF1 quest item*/
+            if (!Core.CheckInventory("Aeacus Empowered", 50))
+                Core.SellItem("Aeacus Empowered", 0, true);
+            ArmyHunt("judgement", new[] { "Ultra Aeacus" }, "Aeacus Empowered", ClassType.Solo, false, 50);
 
-        if (Loot == null)
-        {
-            foreach (string Reward in Loot)
-            {
-                ItemBase item = Bot.Inventory.GetItem(Reward);
-                if (item.Coins)
-                    Core.AddDrop(Reward);
-                else
-                    Core.Logger($"{item} has been excluded as it is not a AC item.");
-            }
-        }
-        else Core.AddDrop(Loot);
+            Core.EquipClass(ClassType.Farm);
+            Adv.BestGear(GearBoost.dmgAll);
+            if (!Core.CheckInventory("Tethered Soul", 300))
+                Core.SellItem("Tethered Soul", 0, true);
+            Core.KillMonster("revenant-999999", "r2", "Left", "*", "Tethered Soul", 300, false); //Temp fix for players > 3
+            //ArmyHunt("revenant",  new[] {"Forgotten Soul"}, "Tethered Soul", false, 300);
 
-        Core.EquipClass(ClassType.Farm);
-        Core.RegisterQuests(questID);
+            if (!Core.CheckInventory("Darkened Essence", 500))
+                Core.SellItem("Darkened Essence", 0, true);
+            ArmyHunt("shadowrealmpast", new[] { "Pure Shadowscythe, Shadow Guardian, Shadow Warrior" }, "Darkened Essence", ClassType.Farm, false, 500);
 
-        Army.SmartAggroMonStart(map, Monsters);
+            Bot.Quests.UpdateQuest(2061);
+            Adv.BestGear(GearBoost.Undead);
+            if (!Core.CheckInventory("Dracolich Contract", 1000))
+                Core.SellItem("Dracolich Contract", 0, true);
+            ArmyHunt("necrodungeon", new[] { "5 Headed Dracolich" }, "Dracolich Contract", ClassType.Farm, false, 1000);
 
-        while (!Bot.ShouldExit)
-            Bot.Combat.Attack("*");
-        Army.AggroMonStop(true);
+            Bot.Drops.Pickup("Revenant's Spellscroll");
```

Comment:
just do a Core.AddDrop(if it isnt in one already,) and a Bot.Wait.ForPickup("item"); at the end

Reply:
Fixed